src/rust/catmem/futures/push.rs

```diff
@@ -36,7 +35,7 @@ pub async fn push_coroutine(ring: Rc<SharedRingBuffer<u16>>, buf: DemiBuffer, yi
                 },
             }
         }
-        trace!("data written ({:?}/{:?} bytes)", index, buf.len());
-        return Ok(());
     }
+    trace!("data written ({:?} bytes)", buf.len());
```

Comment:
Could you leave the error message as it was before? It enables us to check from the logs that we have written all the bytes that were passed on.

Reply:
We no longer have an index to print.

Reply:
Yes, I got that. I am asking you to print `buf.len()` twice. As it is currently, from the logs one cannot easily say how many bytes were written (out of how many).

Reply:
I fixed this.